vignettes/task_plans.Rmd

```diff
@@ -73,15 +73,23 @@ step1 <- create_task_step(step_name = 'prep')
 
 # step2 creates recipes like this:
 # WI_plot.png:
-#   command: visualize(target_name, 'WI_prep', I('Madison'))
+#   command: visualize(
```

Comment:
Should we delete the commented out lines?

Reply:
Unless they're inaccurate, I think these comments provide useful, proximate explanation of what the `create_task_step()` calls do.

Reply:
I kind of like the example of what each step will produce before it is run so I'm for keeping in the commented lines.

Reply:
(and I just checked...I think they're still accurate)